build.gradle

```diff
@@ -209,8 +209,8 @@ project.ext.externalDependency = [
     'wire': 'com.squareup.wire:wire-compiler:3.7.1',
     'charle':  'com.charleskorn.kaml:kaml:0.53.0',
     'common': 'commons-io:commons-io:2.7',
-    'jline':'jline:jline:1.4.1',
     'jetbrains':' org.jetbrains.kotlin:kotlin-stdlib:1.6.0'
+    'pac4jCore': 'org.pac4j:pac4j-core:5.2.0'
```

Comment:
Adding this here only to exclude it in the place where it is used (datahub-frontend) doesn't quite make sense. Even if it wasn't excluded, upgrading to 5.x will require updates to the datahub-frontend code. We cannot update this without those changes in the frontend and testing around the oidc authentication flow.

Reply:
We observed that the existing 'pac4j': 'org.pac4j:pac4j-oidc:4.5.7'  dependency in the source code is defined in datahub main build.gradle so for making code consistent, we did code changes for 'org.pac4j:pac4j-core:5.2.0' in  datahub main build.gradle . If you want to make all the changes in datahub frontend only, we can do that as well.